crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -965,7 +1012,15 @@ impl<E: Backoff> EndpointConnectivityCheckState<E> {
                     }
                 }
             }
-            _ => {}
+            EndpointState::Published
+            | EndpointState::Disconnected(Event::SendCallMeMaybeRequest)
+            | EndpointState::Disconnected(Event::ReceiveCallMeMaybeResponse)
+            | EndpointState::Disconnected(Event::Publish) => {
+                // nothing to do here.
+            }
+            EndpointState::Paused => {
+                // endpoint is paused, meaning it should be dormant, thus not do anything
+            }
```

Comment:
Good question. The answer is it doesn't at all, with one exception - initially `_ =` ate up the Paused event which is nice since it still worked as expected but I was totally unaware of this location. Thus as a matter of personal taste I deconstructed the umbrella `_` into distinct cases for easier referencing and better visibility :)

Also I think it's nice to mention that `Paused` is valid here but we don't need to do anything.